suite-native/navigation/src/components/Screen.tsx

```diff
@@ -70,6 +72,7 @@ export const Screen = ({
         <View
             style={applyStyle(screenContainerStyle, {
                 backgroundColor,
+                paddingTop: header ? 0 : Math.max(insets.top, customVerticalPadding),
```

Comment:
move this `if` logic to style itself

Reply:
https://github.com/trezor/trezor-suite/pull/7753/commits/122aab36e65d7592983793db5ea8f0156040be39